### PR TITLE
Fix legacy multimedia export

### PR DESCRIPTION
**Pull request by benrudolph**
@czue @NoahCarnahan this'll properly match the schema's index to the question id (instead of the export's display value which can change)

http://manage.dimagi.com/default.asp?238002

cc: @proteusvacuum

### CODE REVIEW

File: corehq/apps/reports/tasks.py [hunk 1]
```diff
@@ -402,9 +414,9 @@ def _get_export_properties(export_id, export_is_legacy):
         if export_is_legacy:
             schema = FormExportSchema.get(export_id)
             for table in schema.tables:
-                # - in question id is replaced by . in excel exports
-                properties |= {c.display.replace('.', '-') for c in
-                               table.columns if c.display}
+                properties |= _convert_legacy_indices_to_export_properties(
+                    map(lambda column: column.index, table.columns)
```
Comment (benrudolph) [inline corehq/apps/reports/tasks.py]: main change here is that we're using `column.index` instead of `column.display`

**Comment by dannyroberts** [comment]
Not sure if related:

```======================================================================
FAIL: test_permissions_for_activation (corehq.apps.domain.tests.test_domain_transfer.TestTransferDomainViews)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/domain/tests/test_domain_transfer.py", line 167, in test_permissions_for_activation
    self.assertEqual(resp.status_code, 200)
AssertionError: 404 != 200
-------------------- >> begin captured logging << --------------------
auditcare.middleware: DEBUG: Middleware is running in a test context, disabling monkeypatch
--------------------- >> end captured logging << ---------------------
----------------------------------------------------------------------
```

**Comment by benrudolph** [comment]
fortunately (or unfortunately) i don't think it is. been breaking on other tests

**Review by NoahCarnahan — APPROVED** [review]
👍 on ✅

**Pull request merged**